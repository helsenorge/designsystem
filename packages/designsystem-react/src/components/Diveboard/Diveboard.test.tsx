import React from 'react';
import {render} from '@testing-library/react';
import {DiveBoard} from './DiveBoard';

test('renders default variant correctly', (): void => {
  const {container} = render(<DiveBoard></DiveBoard>);
  expect(container).toMatchSnapshot();
});
