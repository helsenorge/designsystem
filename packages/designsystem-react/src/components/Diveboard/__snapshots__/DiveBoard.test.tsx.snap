// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders default variant correctly 1`] = `
<div>
  <h1>
    Diveboard test
  </h1>
</div>
`;
